app/templates/customer/about.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'customer/customer_base.html' %}
+
+{% block title %}About{% endblock %}
+
+{% block content %}
+<div class="container-fluid bg-white p-3" style="min-height: calc(100vh - 105px);">
+    <div class="p-3 d-flex align-items-center justify-content-center">
+        <div class="col-12 col-md-8 col-lg-8 col-xl-6">
+            <center class="my-5"><span class="p-3 h2 bg-white rounded">😃 About Us</span></center>
+            <li class="list-group-item">Crafts are selected and kits are assembled by Kendra Schreur, with the assistance of her mom, Sharon.</li>
+            <li class="list-group-item">This venture is a way for Kendra to use her God given gifts and have a sense of purpose, especially amidst COVID restrictions.</li>
+            <li class="list-group-item">We hope you and the child in your life will have fun doing these crafts together!</li>
+        </div>
+    </div>
+    <div class="p-3 d-flex align-items-center justify-content-center">
+        <div class="col-6 col-md-4 col-lg-4 col-xl-3">
+        <center><img class="person mb-4 mx-2" src="https://scontent-ort2-2.xx.fbcdn.net/v/t1.6435-9/79231713_1686988968098019_7593991857139351552_n.jpg?_nc_cat=101&ccb=1-3&_nc_sid=09cbfe&_nc_ohc=g_nBCNQWecAAX8SrvbX&_nc_ht=scontent-ort2-2.xx&oh=765a6ef55b271a723416bd9a81ab42bb&oe=60A341D4" alt=""></center>
+        <center><b>Kendra Schreur</b></center>
+        </div>
+        <div class="col-6 col-md-4 col-lg-4 col-xl-3">
+        <center><img class="person mb-4 mx-2" src="https://scontent-ort2-2.xx.fbcdn.net/v/t1.6435-9/134126892_10222353708525288_4443921623824545086_n.jpg?_nc_cat=110&ccb=1-3&_nc_sid=09cbfe&_nc_ohc=NDQT8s6wqDIAX9URrbH&_nc_ht=scontent-ort2-2.xx&oh=a8738e5d8574d5bd321dcb54555f6de2&oe=60A216A6" alt=""></center>
+        <center><b>Sharon Schreur</b></center>
+        </div>
+    </div>
+        <div class="p-3 d-flex align-items-center justify-content-center">
+        <div class="col-12 col-md-8 col-lg-8 col-xl-6">
+            <center class="my-5"><span class="p-3 h2 bg-white rounded">Social 👋</span></center>
+            <center>
+                <a target="_blank" href="https://www.facebook.com/Kids-Krafts-166612741680617" class="facebook"><i class="fa fa-facebook-square m-3 fa-4x"></i></a>
+                <a target="_blank" href="" class="instagram"><i class="fa fa-instagram m-3 fa-4x"></i></a>
+                <a target="_blank" href="" class="pinterest"><i class="fa fa-pinterest-square m-3 fa-4x"></i></a>
```

Comment:
@ethanschreur I recommend you include links for Instagram and Pinterest.

Reply:
> @ethanschreur I recommend you include links for Instagram and Pinterest.

No links exist yet since those social pages do not exist. However, they will exist by the time the website is ready. 
Do you recommend getting rid of them until the social pages are created?

Reply:
Let us remove them since they are not active yet.